README.md

```diff
@@ -16,3 +17,15 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+Proof: 
+
+Let $c$ be any positive integer, $n_0$ be any integer, and both $f(n)$ and $g(n)$ be any positive functions.
+
+Suppose that for all $n \ge n_0$, $f(n) < cg(n)$.
+
+By definition of little o notation, $f(n) \in o(g(n))$.
+
+To meet the definition of big O notation, there must exist positive constants $c$ and $n_0$ such that $0 \le f(n) \le cg(n)$ for all $n \ge n_0$. 
```

Comment:
...and?

Reply:
On another note, any idea why the inline math notation might not be displaying properly in some spots? I am having the same issue on some of the other assignments